libs/application/templates/reference-template/src/fields/Overview/Overview.tsx

```
@@ -0,0 +1,67 @@
+              Values from the table repeater
+            </Text>
+            {tableRepeaterAnswers &&
+              tableRepeaterAnswers.map((ans, i) => {
+                return (
+                  <KeyValue
+                    key={i}
+                    label={`Value ${i + 1}`}
+                    value={`${ans.fullName} - ${ans.nationalId} - ${ans.relation}`}
+                  />
+                )
+              })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)